### PR TITLE
Implement sentry

### PR DESCRIPTION
**Pull request by TorrentofShame**
Implements Sentry into the app and creates a custom transaction called `submitHacker` that logs the registration form and connects to the relevant transaction for the backend.

Test transaction can be seen [here](https://sentry.io/organizations/knight-hacks/discover/hackathon-frontend:3fc015aaa9c545c68154c291d5053ec8/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&project=5898074&query=&sort=-timestamp&statsPeriod=24h&user=926170&yAxis=count%28%29)

### CODE REVIEW

**Review by ailpon — APPROVED** [review]
good to merge after conflict resolve

**Comment by TorrentofShame** [comment]
Cannot merge as of yet, the app is not connecting to sentry, might be an issue with the app not receiving env vars from CF pages

**Comment by TorrentofShame** [comment]
Branch works with cloudflare pages now, good to merge.

**Pull request merged**
